src/plone/restapi/services/auth/renew.py

```diff
@@ -32,6 +32,12 @@ def reply(self):
                          plone.protect.interfaces.IDisableCSRFProtection)
 
         mtool = getToolByName(self.context, 'portal_membership')
+        if (bool(mtool.isAnonymousUser())):
+            # Don't generate authentication tokens for anonymous users.
+            self.request.response.setStatus(401)
+            return dict(error=dict(
+                type='Invalid authentication token',
```

Comment:
@thet should we say here "Invalid or expired authentication token" and in the message "The provided authentication token is invalid or expired."

Reply:
@thet FYI: I fixed those minor issues myself in master.